Nautono/RTOS_framework_v1/src/sail_anglesensor.h

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
You should add a reference to the arduino library if you ended up porting the code.

What would Dr Hughes say ;)